### ISSUE TITLE
Upload 2018 application form

### ISSUE THREAD

**Issue by KimPong** [volunteer-manage/Strawberry-Fields-2018#1]
[Volunteer Application form.docx](https://github.com/volunteer-manage/Strawberry-Fields-2018/files/2172457/Volunteer.Application.form.docx)

@jarvisjohnson 

Here's the app form - open to any edits!

Main updates are inclusion of arrival and departure dates, and larger warnings above credit card section.

**Comment by KimPong** [comment]
Hey @jarvisjohnson 
I've sent Tara the website text to make the volunteers page on the website. 
What do you need in terms of linking the portal to that page?
Is there anything Tara can get ready for you while we are finishing of the form?
xx

**Comment by KimPong** [comment]
@jarvisjohnson 
Can we please change $15 to $18 in the Financial Details section of the application form?

**Comment by jarvisjohnson** [comment]
Yes @KimPong  and you can edit that [here](https://strawberry-fields.shyftly.com/admin/settings)

The application form for testing is [here](url)

I know the facebook login does not work right now, so need to fix that

I haven't made all of the changes in the form you want - when I built Shyftly, I did it based on last years form, and this new one is basically the same but in a different order.  I have stuck with last years one and added in the arrival and leaving dates.   You will still get all of the information you need though.  Let me know if you see any issues with the form though.

Basically, the form is pretty hard coded at this point.  There isn't a way for you to chose what is in the form.  Hopefully will be able to get to a point in the future where you could customise it yourself

**Comment by KimPong** [comment]
Hey @jarvisjohnson thanks for your updates on this - i just did a test and the form is looking great.
found the bit where i can edit the agreement and the charges etc - that's awesome!

i'm really glad to have the arrival/leaving dates.
i can run with not having the other changes I requested if i can stipulate what goes in the offer email we send to people when we accept them. 
but once apps are launched we will have a month to work on things like that.
Is there any way that we can at least add the check box for people who have medical certificates? Ruby wanted that so she could view who had that qualification without reading all the apps.

Can I clarify that the 3 links from the application home page for the different vol types all point to the same form?
I'll just need to assure people that they can apply for more than one position, whichever one they pick. 

also just fyi, i'll still be adding jobs within the 'General' Section, such as Gate, Traffic, Waste ec when it couls to assigning shifts.

and Tara is uploading the vols page on the website, so I'll get testing done by a couple of people and get this up asap.

**Comment by KimPong** [comment]
@jarvisjohnson is the text in the terms something that you put together? it covers things really well, just want to know the source. x

**Comment by KimPong** [comment]
testing feedback:
1. under Buddies: edit text - changes in bold.
 **T**his is for organisational purposes and will not affect your application. **Shift requests cannot be guaranteed**.

2. Uploading documents. When I click to choose a doc it only displays docs on my computer that are images. Not PDFs for example. Is that just my computer do you think?

3. In the admin back end, When I click to view an application I get a blank page. This is the url that comes up if that helps: https://strawberry-fields.shyftly.com/applies/cd6cbb22-4b1a-48b8-9c5b-132bfffa3bcb

**Comment by KimPong** [comment]
Heya again @jarvisjohnson 
Tara has made this page on the website and she wants to launch on Thursday - so your Thurs morning or Wed evening.
https://www.strawberry-fields.com.au/volunteer-applications 
Password is kimpong
does she just need to embed this link? https://strawberry-fields.shyftly.com

The only other thing I wanted to check was whether the site will recognise emails/profiles from last year? 
Ruby tried to create a profile but was receiving the 'oops something went wrong' error page. 
When she tried with a different email address it worked. 

If people who have volunteered before should go to 'My Profile' instead of 'Apply to volunteer' I would add a line about that into the web copy. But maybe the error was caused by something else? 
xx

**Comment by jarvisjohnson** [comment]
@KimPong - think i've covered it all below

- Yes, all of the links go to the same form - perhaps that is confusing though  - let me know if you want to get rid of the buttons.

- Yes, the Shyftly terms linked to in the footer are something I wrote.

 - Testing
   1. Ok will change the text

  2. Will allow pdfs to be uploaded

  3. I need to fix the link in the admin.  There should be a '/admin' in the url, so that link above should be https://strawberry-fields.shyftly.com/admin/applies/cd6cbb22-4b1a-48b8-9c5b-132bfffa3bcb

Yes, Tara can use that link.  We can set to site to be volunteers.strawberry-fields.com.au is she wants, she will, like last year, need to update the DNS, with a CNAME pointing to strawberry-fields.shyftly.com


I didn't end up importing the users from last year sorry.  It is going to be a lot of work, as the site has changed quite a bit so the data won't match up now.

I don't think Ruby's issue is related to that. I got a notification of the error so I will fix it. I've seen a couple of errors come in during your testing so I will take care of them.

The 'apply to volunteer' button should disappear once an application is submitted - so all they can do is see their profile, there won't be anything special for people from last year.

I will get these things sorted over the weekend

**Comment by jarvisjohnson** [comment]
Oh on the above, if Tara is happy having the url be strawberry-fields.shyftly.com, then that's easy for me - and yes she would just need to change that link in here page as you've said

**Comment by KimPong** [comment]
@jarvisjohnson 
thanks i'll let Tara know those two options.
let's not worry about getting last year's vol data back. 
I have an export from last year so i at least have a list of who was confirmed. 
i can look at who selects the previous volunteer option and cross check 👍

**Comment by jarvisjohnson** [comment]
Ok @KimPong all of the above issues should be resolved now.

Let me know if you are able to test submitting an application.

You should see the person in the Stripe account once they have submitted their application
